include/algorithms/decision_forest/decision_forest_classification_predict_types.h

```diff
@@ -105,8 +115,26 @@ class DAAL_EXPORT Input : public classifier::prediction::Input
     services::Status check(const daal::algorithms::Parameter * parameter, int method) const DAAL_C11_OVERRIDE;
 };
 
+/**
+ * <a name="DAAL-STRUCT-ALGORITHMS__CLASSIFIER__PARAMETER"></a>
+ * \brief Class for the parameters of the Decision Forest classification algorithm
+ *
+ * \snippet decision_forest/decision_forest_classification_predict_types.h Parameter source code
+ */
+/* [interface1::Parameter source code] */
+struct DAAL_EXPORT Parameter : public daal::algorithms::classifier::interface2::Parameter
+{
+    Parameter(size_t nClasses = 2) :
```

Comment:
default value for nClasses is inconsistent - some work to delete it was performed some time ago - double check please. Parameter for viting is absent